pkg/controller/clone-controller.go

```diff
@@ -548,8 +548,10 @@ func MakeCloneSourcePodSpec(sourceVolumeMode corev1.PersistentVolumeMode, image,
 			Name:      cloneSourcePodName,
 			Namespace: sourcePvcNamespace,
 			Annotations: map[string]string{
-				cc.AnnCreatedBy: "yes",
-				AnnOwnerRef:     ownerRefAnno,
+				cc.AnnCreatedBy:                  "yes",
+				AnnOwnerRef:                      ownerRefAnno,
+				cc.AnnPodSidecarInjectionIstio:   sourcePvc.Annotations[cc.AnnPodSidecarInjectionIstio],
```

Comment:
You should be able to use `SetPvcAllowedAnnotations` for passing annotations from the data volume to the transfer pods

Reply:
Yeah - both prep-claim and the size detection pod were missing this.

Reply:
Yeah looks much better now